### ISSUE TITLE
[Indentation] not handled correctly when property is prefixed by *

### ISSUE THREAD

**Issue by onigoetz** [stylelint/stylelint#358]
Hi,

it seems the indentation doesn't check correctly when I use `*` hacks with the property name

```
rules: {
    "indentation": [2, 4],
}	
```

```css
.Button {
    position: relative;

    display: inline-block;
    *display: inline;
    zoom: 1;

    *border: 0;
    border-width: 4px;
    border-style: solid;
    border-radius: 4px;
}
```

```
input.css
5:5	⚠  Expected indentation of 4 spaces at line 5 (indentation) [stylelint]
8:5	⚠  Expected indentation of 4 spaces at line 8 (indentation) [stylelint]
```

**Comment by jeddy3** [comment]
Thanks for reporting this.

@davidtheclark Perhaps something to do with `*` being in the `raws` and thus the property name is considered to be indented by one space to many?

**Comment by davidtheclark** [comment]
Yes, you're right @jeddy3 : PostCSS is putting the `*` in `raws.before`. A little change should fix it.
